### ISSUE TITLE
Android TalkBack reads elements behind modal pages

### ISSUE THREAD

**Issue by BenBtg** [xamarin/Xamarin.Forms#3622]
### Description
Android TalkBack reads contents of parent page when Modal page is navigated to.

### Steps to Reproduce

1. Run the ModalNavigation Forms Sample on Droid
2. Enable TalkBack Accessibility feature on the device
3. Swipe right to read out each item in list. (Works as expected)
4. Navigate to an item by double tapping
5. Swipe right to read out the  modal page

### Expected Behavior
Contents of visible page is read out

### Actual Behavior
Contents of parent view is read out and each item in list is highlighted even though it is not visible.

### Basic Information

- Version with issue: 3.2
- Last known good version: Could not find a working version. 2.5 also has issue
- IDE: VS for Mac
- Platform Target Frameworks: 
  - Android: Tested on 5.0, 7.0, 9.0
- Nuget Packages: Forms
- Affected Devices: All Droid. Tested on Moto G, Pixel 2 and SGS 7

### Reproduction Link

https://github.com/xamarin/xamarin-forms-samples/tree/master/Navigation/Modal

**Comment by hartez** [comment]
See also #3612.

**Comment by BenBtg** [comment]
Any update on this issue?

**Comment by yuri-kulikov** [comment]
Any update, ETA or maybe at least some workaround?

**Comment by kspearrin** [comment]
Also seeing this issue. Is there a workaround?

**Comment by mikegottlieb** [comment]
I haven't found any workaround to this or any of the other huge number of problems I've seen trying to make TalkBack and XF work well together. I followed up a bit in the Xamarin Slack channel about it, but it seems unlikely much will get done any time soon. Also, it will require me to upgrade XF versions which is itself likely to break much more than it might fix.

I wrote back to the user of our app who reported the issue that there was nothing I could do and that unfortunately they will have to stick to our website.
